### DISCUSSION TITLE
建议作者为了展示本扩展效果，将示例视频换成这个

### DISCUSSION THREAD

**Discussion by yuijker** [hanydd/BilibiliSponsorBlock#91] [Ideas]
建议更换示例视频为
【官方双语】评 测 视 频（基本不含赞助内容）#linus谈科技
 https://www.bilibili.com/video/BV1bY4y1v7Mb/
对应YouTube原链接：
This Review Gets Stranger and Stranger (April Fools 2022)
https://youtu.be/Slwai2Jhy_A?si=Az0_fnBbOHt28nhV

**Comment by hanydd** [comment]
合理，这视频就离谱
